datasets/cfpb_complaints/_images/run_csv_transform_kub/requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
+pandas
+google-cloud-storage
```

Comment:
Please add `numpy` here. And arrange this alphabetically

Reply:
added numpy